src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -425,6 +427,12 @@ def set_survey_info(show_survey_message):
     _session.show_survey_message = show_survey_message
 
 
+@decorators.suppress_all_exceptions()
+def set_wam_info(allow_broker):
```

Comment:
How about we just call it `broker` to align with the config name? WAM is a Windows-only concept.